adafruit_esp32spi/adafruit_esp32spi_socket.py

```diff
@@ -148,6 +151,39 @@ def settimeout(self, value):
         """Set the read timeout for sockets, if value is 0 it will block"""
         self._timeout = value
 
+    def available(self):
+        """Returns how many bytes of data are available to be read (up to the MAX_PACKET length)"""
+        if self.socknum != NO_SOCKET_AVAIL:
+            return min(_the_interface.socket_available(self._socknum), MAX_PACKET)
+        return 0
+
+    def connected(self):
+        """Whether or not we are connected to the socket"""
+        if self.socknum == NO_SOCKET_AVAIL:
+            return False
+        elif self.available():
+            return True
+        else:
+            status = _the_interface.socket_status(self.socknum)
+            # TODO: why is esp.<ConstantName> not defined? using magic numbers in mean time
```

Comment:
I wanted to use the same constants as defined in `adafruit_esp32spi`, but for some reason I wasn't able to get that to work. The constants weren't accessible off of the that module when running in CP (but my editor was showing them as available)

Any ideas? We obviously don't want the magic numbers here.

Reply:
Which line are you referring to here?

Reply:
all of the numbers in `status not in (magic numbers)`

All of those are constants corresponding to ones defined in esp32spi.py https://github.com/adafruit/Adafruit_CircuitPython_ESP32SPI/blob/master/adafruit_esp32spi/adafruit_esp32spi.py#L101-L111

My initial attempt to use them wasn't working, I'm sure there is just something silly i'm missing with being able to re-use those constants here.